ext/crates/fp/src/vector/specialized.rs

```diff
@@ -0,0 +1,502 @@
+//! This module replaces `specialized` when `odd-primes` is disabled. Instead of producing enum
+//! wrappers, it simply rexports `FooP<2>` as `Foo`.
```

Comment:
I assume this is supposed to go to specialized_2.rs?

Reply:
Thanks!